src/sentry/runner/initializer.py

```diff
@@ -187,6 +188,16 @@ def configure_structlog():
 
     structlog.configure(**kwargs)
 
+    lvl = os.environ.get('SENTRY_LOG_LEVEL', 'INFO')
+    if lvl in logging._levelNames:
```

Comment:
Can we just use the same tuple of `LOG_LEVELS` we use for validating CLI input against? That way we're not leveraging this private `_levelNames`? Probably not a big deal, but anything from stdlib that starts with `_`, I'm hesitant to use.